src/main/kotlin/org/digma/intellij/plugin/ui/list/insights/DurationBreakdownPanel.kt

```diff
@@ -182,39 +177,39 @@ private fun getSpanDisplayNameLabel(
         moreData: HashMap<String, Any>,
 ): JComponent {
     val spanId = CodeObjectsUtil.createSpanId(durationBreakdown.spanInstrumentationLibrary, durationBreakdown.spanName)
-    val displayName = durationBreakdown.spanDisplayName
+    val fullDisplayName = durationBreakdown.spanDisplayName
+    var limitedDisplayName = fullDisplayName
+    if (fullDisplayName.length > 30) {
+        limitedDisplayName = fullDisplayName.trim().take(30) + "..."
+    }
```

Comment:
maybe taken it out to common function, so it could be reused?

Reply:
sure! I will do that!